Improve fdas (#236)
telefonicaid/fiware-data-access: doc/03_api.md

| ------------------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
| `defaultDataAccess` | ✓        | Overrides the instance default and enables or disables automatic `defaultDataAccess` creation for this FDA. The default value is taken from `FDA_CREATE_DEFAULT_DATA_ACCESS`; if that env var is not set, the default is `true`. | `false` |

> When `validationMode="unchecked"` is sent in the request body, synchronous validation is skipped and initial

Reviewer: ```suggestion
When `validationMode` is set to `unchecked` in the request body, synchronous validation is skipped and initial
```

The payload is a JSON object containing a FDA that follows the JSON FDA representation format (described in
[FDA payload datamodel](#fda-payload-datamodel) section).

> `validationMode="unchecked"` is useful for heavy queries where synchronous validation could delay creation. In this mode

Reviewer: ```suggestion
`validationMode=unchecked` is useful for heavy queries where synchronous validation could delay creation. In this mode
```

As this is the way to be used in a query param, isn't it?

Reviewer: Ok, it is not a query param... so my suggestion would be:

```suggestion
`validationMode` set to `"unchecked"` is useful for heavy queries where synchronous validation could delay creation. In this mode
```

Author: Yes

The payload is a JSON object containing a FDA that follows the JSON FDA representation format (described in
[FDA payload datamodel](#fda-payload-datamodel) section).

> `validationMode=unchecked` is useful for heavy queries where synchronous validation could delay creation. In this mode

Reviewer: ```suggestion
`validationMode` set to `unchecked` is useful for heavy queries where synchronous validation could delay creation. In this mode
```
